framework/packages/abstract-integration-tests/src/mock_modules.rs

```diff
@@ -13,6 +13,7 @@ use cw_orch::prelude::*;
 pub type MockAdapterContract = AdapterContract<AdapterMockError, Empty, Empty, Empty, Empty, Empty>;
 pub type MockAppContract = AppContract<AppMockError, Empty, Empty, Empty, Empty, Empty, Empty>;
 
+#[allow(ambiguous_glob_reexports)]
```

Comment:
I would prefer to not allow it

Reply:
Everything is re-exported that's in there so there shouldn't be any reason why you'd need to index into it (which is the only way this could be an issue) 

If you still not like it I can see about changing the mod names or even removing the internal mods.

Reply:
I might be mis-understanding it. Investigating...

Reply:
TY for fixing!